go-controller/pkg/util/kube.go

```diff
@@ -66,6 +66,8 @@ func newKubernetesRestConfig(conf *config.KubernetesConfig) (*rest.Config, error
 	if err != nil {
 		return nil, err
 	}
+	kconfig.QPS = 25
+	kconfig.Burst = 25
```

Comment:
are those the recommended values?

Reply:
Those values were recommended by OpenShift's scalability team, which had tested them and seen pod ready latency reach "normal values": see, https://bugzilla.redhat.com/show_bug.cgi?id=1901363#c5

Reply:
thanks